extensions/eclipse-che-theia-plugin-ext/src/node/plugin-service.ts

```diff
@@ -46,22 +46,23 @@ export class PluginApiContributionIntercepted extends PluginApiContribution {
         const pluginExtModulePath = path.dirname(require.resolve('@theia/plugin-ext/package.json'));
         const webviewStaticResources = path.join(pluginExtModulePath, 'src/main/browser/webview/pre');
 
-        this.cheApi.findUniqueServerByAttribute(SERVER_TYPE_ATTR, SERVER_IDE_ATTR_VALUE).then(server => {
-            let domain;
-            if (server.url) {
-                domain = getUrlDomain(server.url);
-            }
-            const hostName = this.handleAliases(process.env[WebviewExternalEndpoint.pattern] || domain || WebviewExternalEndpoint.pattern);
-            webviewApp.use('/webview', serveStatic(webviewStaticResources));
-
-            console.log(`Configuring to accept webviews on '${hostName}' hostname.`);
-            app.use(vhost(new RegExp(hostName, 'i'), webviewApp));
-
-            this.waitWebviewEndpoint.resolve();
-        }).catch(err => {
-            console.log('Unable to configure webview domain: ', err);
-            this.waitWebviewEndpoint.resolve();
-        });
+        this.cheApi.findUniqueServerByAttribute(SERVER_TYPE_ATTR, SERVER_WEBVIEWS_ATTR_VALUE).then(server => {
+                let domain;
+                if (server.url) {
+                    domain = getUrlDomain(server.url);
+                }
+                const hostName = this.handleAliases(process.env[WebviewExternalEndpoint.pattern] || domain || WebviewExternalEndpoint.pattern);
+                webviewApp.use('/webview', serveStatic(webviewStaticResources));
+    
+                console.log(`Configuring to accept webviews on '${hostName}' hostname.`);
```

Comment:
Maybe it would be better to use the logger? How it was done in [hosted-plugin-remote.ts#L30-L31](https://github.com/eclipse/che-theia/blob/master/extensions/eclipse-che-theia-plugin-remote/src/node/hosted-plugin-remote.ts#L30-L31)
```
import { ILogger } from '@theia/core/lib/common';
...
@inject(ILogger)
protected readonly logger: ILogger;
...
this.logger.debug('...');
```

Reply:
Thanks for noticing this.
Updated